modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -0,0 +1,21 @@
+using gapi_GKernelPackage = cv::gapi::GKernelPackage;
```

Comment:
If this file can have comments, having a comments here would help

Reply:
also, does it need the include guards, license header, or something?

Reply:
It's a good question     
I suppose this file used only by python wrapper and it isn't necessary to have include guards    
Check this: https://github.com/opencv/opencv/blob/master/modules/dnn/misc/python/pyopencv_dnn.hpp

@dkurt Could you comment it ?

Reply:
Added comment